src/index.ts

```diff
@@ -69,6 +69,14 @@ export const eventHandler = async (
   // bg: "ffffff",
   // u_nplug: 3
   // u_nmime: 4
+
+  if (settings.conversionLinker) {
+    // always handle conversion linker if gclid query param exists
```

Comment:
always or only? what's the intention here?
```suggestion
    // only handle conversion linker if gclid query param exists
```